modules/update-network-sysctl-allowlist.adoc

```diff
@@ -0,0 +1,222 @@
+  namespace: openshift-multus
+  resourceVersion: "2422"
+  uid: 96d138a3-160e-4943-90ff-6108fa7c50c3
+----
+
+. Edit the list by using the following command:
++
+[source,terminal]
+----
+$ oc edit cm -n openshift-multus cni-sysctl-allowlist -oyaml
+----
++
+For example, to allow you to be able to implement stricter reverse path forwarding you need to add `^net.ipv4.conf.IFNAME.rp_filter$` and `^net.ipv6.conf.IFNAME.rp_filter$` to the list as shown here:
++
+[source,terminal]
+----
+# Please edit the object below. Lines beginning with a '#' will be ignored,
+# and an empty file will abort the edit. If an error occurs while saving this file will be
+# reopened with the relevant failures.
+#
+apiVersion: v1
+data:
+  allowlist.conf: |-
+    ^net.ipv4.conf.IFNAME.accept_redirects$
+    ^net.ipv4.conf.IFNAME.accept_source_route$
+    ^net.ipv4.conf.IFNAME.arp_accept$
+    ^net.ipv4.conf.IFNAME.arp_notify$
+    ^net.ipv4.conf.IFNAME.disable_policy$
+    ^net.ipv4.conf.IFNAME.secure_redirects$
+    ^net.ipv4.conf.IFNAME.send_redirects$
+    ^net.ipv4.conf.IFNAME.rp_filter$
+    ^net.ipv6.conf.IFNAME.accept_ra$
+    ^net.ipv6.conf.IFNAME.accept_redirects$
+    ^net.ipv6.conf.IFNAME.accept_source_route$
+    ^net.ipv6.conf.IFNAME.arp_accept$
+    ^net.ipv6.conf.IFNAME.arp_notify$
+    ^net.ipv6.neigh.IFNAME.base_reachable_time_ms$
+    ^net.ipv6.neigh.IFNAME.retrans_time_ms$
+    ^net.ipv6.conf.IFNAME.rp_filter$
+----
+
+. Save the changes to the file and exit.
++
+[NOTE]
+====
+The removal of `sysctls` is also supported. Edit the file, remove the `sysctl` or `sysctls` then save the changes and exit.
+====
+
+.Verification
+
+Follow this procedure to enforce stricter reverse path forwarding for IPv4.
+For more information on reverse path forwarding see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security_guide/sect-security_guide-server_security-reverse_path_forwarding[Reverse Path Forwarding
+].
+
+. Create a network attachment definition, such as `reverse-path-fwd-example.yaml`, with the following content:
++
+[source,yaml]
+----
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: tuningnad
+  namespace: default
+spec:
+  config: '{
+    "cniVersion": "0.4.0",
+    "name": "tuningnad",
+    "plugins": [{
+      "type": "bridge"
+      },
+      {
+      "type": "tuning",
+      "sysctl": {
+         "net.ipv4.conf.IFNAME.rp_filter": "1"
+        }
+    }
+  ]
+}'
+----
+
+. Apply the yaml by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f reverse-path-fwd-example.yaml
+----
++
+.Example output
+[source,terminal]
+----
+networkattachmentdefinition.k8.cni.cncf.io/tuningnad created
+----
+
+. Create a pod such as `examplepod.yaml` using the following YAML:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+  labels:
+    app: httpd
+  namespace: default
+  annotations:
+    k8s.v1.cni.cncf.io/networks: tuningnad <1>
```

Comment:
There is a special button for copying the configuration and it copies the configuration with  `<1> `     ` k8s.v1.cni.cncf.io/networks: tuningnad <1>`
So configuration is wrong, is it expected?